Mathlib/Data/NNReal/Basic.lean

```diff
@@ -1268,7 +1271,7 @@ def evalNNRealtoReal : PositivityExt where eval {u α} _zα _pα e := do
     let ra ← core q(inferInstance) q(inferInstance) a
     assertInstancesCommute
     match ra with
-    | .positive pa => pure (.positive q(nnreal_coe_pos $pa))
+    | .positive pa => pure (.positive q(Bound.coe_pos_of_pos $pa))
```

Comment:
What happened here? This seems like unnecessary entanglement between `bound` and `positivity`.

Reply:
Previously `nnreal_coe_pos` was a private lemma in this scope. I also needed it for `bound`, so I pulled it out to a toplevel thing and called it `NNReal.coe_pos_of_pos`. Then I was told to rename it to `Bound.coe_pos_of_pos` by @YaelDillies. I did this even though it has this one other use, in part because the use is private and within the same file.

Happy to either re-add the `nnreal_coe_pos` alias (then we would have two names for the same lemma, for two different tactics), or undo the `Bound.` scoping, or do nothing.

Reply:
Another options would be to use `coe_pos.mpr $pa` here, at the expense of producing a very-slightly-larger proof term.

Reply:
Ah I see what happened here. My opinion is that if two tactics want to use the same non-existing lemma, then both of them should define that lemma (possibly using `alias` from an existing lemma). The point is to make sure that tactics get precisely the lemma they want, avoiding people who modify non-tactic lemmas having to figure out how to fix the tactics.

So here you should just not touch the `positivity`-specific lemma and make your own `Bound`-specific one (which can henceforth be private, 🎉)

Reply:
Fixed, and I made the other `Bound` aliases private as well.